cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterDescriptor.java

```diff
@@ -152,6 +152,34 @@ public void loadProps() {
           .parseInt(properties.getProperty("read_data_consistency_level",
               Integer.toString(conf.getReadDataConsistencyLevel()))));
 
+      conf.setConcurrentQPSubTaskThread(Integer
+          .parseInt(properties.getProperty("concurrent_qp_sub_task_thread",
+              Integer.toString(conf.getConcurrentQPSubTaskThread()))));
+      if (conf.getConcurrentQPSubTaskThread() <= 0) {
+        conf.setConcurrentQPSubTaskThread(Runtime.getRuntime().availableProcessors());
+      }
+
+      conf.setConcurrentRaftTaskThread(Integer
+          .parseInt(properties.getProperty("concurrent_raft_task_thread",
+              Integer.toString(conf.getConcurrentQPSubTaskThread()))));
+      if (conf.getConcurrentRaftTaskThread() <= 0) {
+        conf.setConcurrentRaftTaskThread(Runtime.getRuntime().availableProcessors());
+      }
+
+      conf.setCloseRaftTaskBlockTimeout(Integer
+          .parseInt(properties.getProperty("close_raft_task_block_timeout",
+              Integer.toString(conf.getCloseRaftTaskBlockTimeout()))));
+      if (conf.getConcurrentRaftTaskThread() <= 0) {
+        conf.setCloseRaftTaskBlockTimeout(Runtime.getRuntime().availableProcessors());
+      }
+
+      conf.setCloseQPSubTaskBlockTimeout(Integer
+          .parseInt(properties.getProperty("close_qp_sub_task_block_timeout",
+              Integer.toString(conf.getCloseQPSubTaskBlockTimeout()))));
+      if (conf.getCloseQPSubTaskBlockTimeout() <= 0) {
+        conf.setCloseQPSubTaskBlockTimeout(Runtime.getRuntime().availableProcessors()*10);
```

Comment:
closeRaftTaskBlockTimeout is default as cpu core number * 10?

Reply:
fixed